plugins/linux-v4l2/v4l2-mjpeg.c

```diff
@@ -45,7 +45,7 @@ int v4l2_init_mjpeg(struct v4l2_mjpeg_decoder *decoder)
 	}
 
 	decoder->context->flags2 |= AV_CODEC_FLAG2_FAST;
-	decoder->context->pix_fmt = AV_PIX_FMT_YUVJ422P;
+	// decoder->context->pix_fmt = AV_PIX_FMT_YUVJ422P;
```

Comment:
Would prefer removing this line entirely before merging.

Reply:
Yea forgot to clean that up. Done